CONTRIBUTING.md

```diff
@@ -1,10 +1,37 @@
+- *Bug fix*. Change addresses issues in the specification as raised in at least one of GitHub's Issues with sufficient details.
+- *New content*. Change introduces new sections to the specification, such as a proposed new operation or an additional method of an interface. Similar to a bug fix, a new content change should be accompanied by a GitHub Issue or other publicly-disclosed artifact that details a need and the proposed new content as a remedy.
+
+It is strongly recommended that a pull request represents a change of just one of these stated categories. For instance, it is not advisable for a pull request that contains a bug fix or new content to also accompany a stylistic change to an unrelated section; or a stylistic change that also accompanies some wording changes. Reasonable specific exceptions, such as opportunistically fixing some misspelled words in a limited amount, are acceptable.
+
+It is also highly desirable that a change to address a certain issue is wholly represented within a single pull request. In other words, a change represented by a pull request is atomic and not distributed across multiple pull requests submitted at different times. This guideline is here to reduce the chance that a specification remains in a transient state for an unspecified period while multiple separate pull requests are still under review.
+
+Change to the document structure, as in rearranging sections within the document without changing the content itself, is considered a stylistic change.
+
+## Process details
+Wording change does not require opening a GitHub Issue, as the change in the pull request is self-sufficient. However, wording changes should improve the overall readability and interpretation of the content in an obvious way. Keep in mind that these are subjective qualities, and not everyone may agree to the change. Ultimately, it is up to the consideration of editors whether to accept it.
+
+Similarly, a stylistic change does not necessarily require opening a GitHub Issue. It does, however, require buy-ins from the Working Group in order to proceed. The best way to propose this type of change is to attend one of the bi-weekly Web Machine-Learning Working Group teleconference calls and get a vote. A practical way to reach out to the Working Group to get invited to the teleconference call is to post a GitHub Issue giving a rough explanation of the proposed change and ask to be invited.
+
+Bug fixes and new content changes should proceed as follows:
+1. **Open an issue in GitHub Issues** with a brief description of the problem and a potential solution if it's not already obvious. A proposal or suggestion for improvement may need a bit more explanation with possible references to related information. An active issue is the best way to get attention. Members of the Working Group scan active issues constantly.
```

Comment:
For new operation proposal, should we define a template / check list for contributors to fill? 

The existing [Guidelines for new operations](https://www.w3.org/TR/webnn/#security-new-ops)  would be a good check list from security perspective. Based on the discussion of [Rationale/criteria for adding new ops to the WebNN API](https://www.w3.org/2021/11/18-webmachinelearning-minutes.html#t02), some additional aspects may need to be included in the github issue, e.g.:
- use case
- sample model
- cross-frameworks support
- cross-platforms implementability

Reply:
Please check out the new commit.

Reply:
LGTM, thanks much!